packages/build-tools/src/utils/prebuild.ts

```diff
@@ -46,13 +47,13 @@ function getPrebuildCommand(job: Job): string {
   const expoCommandPrefix = 'expo ';
   const expoCliCommandPrefix = 'expo-cli ';
   if (prebuildCommand.startsWith(npxCommandPrefix)) {
-    prebuildCommand = prebuildCommand.substr(npxCommandPrefix.length).trim();
+    prebuildCommand = prebuildCommand.substring(npxCommandPrefix.length).trim();
   }
   if (prebuildCommand.startsWith(expoCommandPrefix)) {
-    prebuildCommand = prebuildCommand.substr(expoCommandPrefix.length).trim();
+    prebuildCommand = prebuildCommand.substring(expoCommandPrefix.length).trim();
   }
   if (prebuildCommand.startsWith(expoCliCommandPrefix)) {
-    prebuildCommand = prebuildCommand.substr(expoCliCommandPrefix.length).trim();
+    prebuildCommand = prebuildCommand.substring(expoCliCommandPrefix.length).trim();
   }
-  return prebuildCommand;
+  return prebuildCommand.split(' ');
```

Comment:
I don't have strong opinion on that and I don't see any specific way where this would break, but in general split like that is not a safe operation